doc/source/contributing/syntax_guide.md

```diff
@@ -0,0 +1,201 @@
+(syntax_guide)=
+# Syntax Guide
+
+## Adding targets
+Adding custom targets or anchors to the headings is really helpful for cross-referencing. They allow us to link to the heading using a simple syntax.
+They are defined using this syntax:
+
+::::{tab-set}
+
+:::{tab-item} rST
+:sync: rst
+
+```
+.. _mytarget:
+```
+:::
+
+:::{tab-item} MyST (Markdown)
+:sync: myst
+
+```
+(mytarget)=
+```
+:::
+
+::::
+
+They are referred using this syntax:
+
+::::{tab-set}
+
+:::{tab-item} rST
+:sync: rst
+
+```
+:ref:`mytarget`
+```
+:::
+
+:::{tab-item} MyST (Markdown)
+:sync: myst
+
+```
+{ref}`mytarget`
+```
+:::
+
+::::
+
+For adding anchors in `.ipynb` files, Markdown syntax will be used in the markdown cells of `.ipynb` files.
+
+## Backticks for highlighting code keywords
+
+For highlighting inline code keywords or file names, backticks are used. In Markdown single backticks are used while in rST double backticks are used. For example, for highlighting the file name `conf.py`, we will use this syntax:
+
+````{tabbed} rST
```

Comment:
tabbed is from sphinx-panels and we should not use it anymore

Reply:
unresolving this too. panels should not be used anymore and I removed the extension competely in #1923, it doesn't work anymore: https://arviz--1903.org.readthedocs.build/en/1903/contributing/syntax_guide.html#backticks-for-highlighting-code-keywords